hypertrace-alert-engine/src/main/resources/configs/common/application.conf

```diff
@@ -12,16 +12,37 @@ job.config = {
   tenant_id = ${?DEFAULT_TENANT_ID}
 }
 
-ruleSource = {
+notificationRuleSource = {
   type = "fs"
   fs = {
-    path = "build/resources/main/alert-rules.json"
+    path = "build/resources/test/configs/rules.json"
```

Comment:
both the path are pointing to the same file, you can lookup the name of the file in the resource folder